src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+
 public class Duke {
+    // DUKE in ASCII art
+    private static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    // Horizontal Rule to act as a divider
+    private static final String HORIZONTAL_RULE = "__________________________________________________";
+
+    /**
+     * Start up function.
+     * Prints ASCII art and prompts users for input
+     */
+    public static void startUp() {
+        System.out.println(LOGO);
+        System.out.println(HORIZONTAL_RULE);
+        System.out.println("Hello! I'm Duke. \nWhat can I do for you?");
+        System.out.println(HORIZONTAL_RULE);
+    }
+
+    /**
+     * Shut down function.
+     * Informs users that Duke is no longer accepting commands
+     */
+    public static void shutDown() {
+        System.out.println(HORIZONTAL_RULE);
+        System.out.println("Shutting Down! Hope to see you again soon!");
+        System.out.println(HORIZONTAL_RULE);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // tasks for storing tasks + taskCounter to track next empty cell
+        Task[] tasks = new Task[100];
+        int taskCounter = 0;
+
+        // Setting up input reading
+        Scanner s = new Scanner(System.in);
+
+        startUp();
+
+        // Reads inputs until "bye" is sent
+        while (true) {
+            String userInput = s.next();
+            userInput += s.nextLine();
+
+            // Extracts first word in input
+            // Used to check if tasks are to be marked/unmarked
+            String firstWord = userInput.split(" ", 2)[0].toLowerCase();
+
+            if ((userInput).equalsIgnoreCase("bye")) {
+                shutDown();
+                return;
+
+            } else if ((userInput).equalsIgnoreCase("list")) {
+                // Prints all contents of task Array
+                System.out.println(HORIZONTAL_RULE);
+                for (int i = 0; i < taskCounter; i++) {
+                    System.out.println((i + 1) + ". " + "[" + tasks[i].getStatus() + "] " + tasks[i].getTask());
+                }
```

Comment:
It may be good to create a method for this in the task class. Perhaps keep the array of tasks in the Task class and add to it within the constructor